packages/forms-web-app/src/controllers/examination/your-interested-party-number.js

```diff
@@ -0,0 +1,71 @@
+const {
+	routesConfig: {
+		examination: {
+			pages: {
+				yourInterestedPartyNumber: {
+					view: yourInterestedPartyNumberView,
+					route: yourInterestedPartyNumberRoute
+				},
+				haveAnInterestedPartyNumber: { route: haveAnInterestedPartyNumberRoute },
+				submittingFor: { route: submittingForRoute }
+			}
+		}
+	},
+	routesConfig: { examination }
+} = require('../../routes/config');
+
+const config = require('../../config');
+
+const interestedPartyNumberString = 'interested-party-number';
```

Comment:
It would be good if this was set to the id retrieved from the '/routes/config' 'yourInterestedPartyNumber' object. We would then be able to feed that into the view and use it there.